docker-compose.yml

```diff
@@ -93,7 +93,6 @@ services:
         PRODUCT_DEF_BRANCH: ${PRODUCT_DEF_BRANCH}
         PS_CONSTANTS_URL: ${PS_CONSTANTS_URL}
         SNIPPET_CREATION: ${SNIPPET_CREATION}
-        SNIPPET_CREATION_START: ${SNIPPET_CREATION_START}
```

Comment:
Why do we remove this line? The variable would be non-settable for the local dev testing without it

Reply:
Fixed.